src/app/modules/submission/encrypt-submission/__tests__/encrypt-submission.utils.spec.ts

```diff
@@ -101,4 +102,78 @@ describe('encrypt-submission.utils', () => {
       expect(newFilenames).toContain(`2-${validSingleFile.filename}`)
     })
   })
+
+  describe('getPaymentIntentDescription', () => {
+    it('should description for Fixed Payments Type', () => {
+      const expectedValue = 'expectedValue'
+      const formData = {
+        payments_field: {
+          payment_type: PaymentType.Fixed,
+          description: expectedValue,
+          name: 'name',
+        },
+      } as unknown as IPopulatedEncryptedForm
+
+      const products: any = [{}]
+
+      const result = getPaymentIntentDescription(formData, products)
+
+      expect(result).toEqual(expectedValue)
+    })
+    it('should return name for Variable Payment Type', () => {
+      const expectedValue = 'expectedValue'
+      const formData = {
+        payments_field: {
+          payment_type: PaymentType.Variable,
+          description: 'description',
+          name: expectedValue,
+        },
+      } as unknown as IPopulatedEncryptedForm
+
+      const products: any = [{}]
+
+      const result = getPaymentIntentDescription(formData, products)
+
+      expect(result).toEqual(expectedValue)
+    })
+
+    it('should return product names for Products Payment Type', () => {
+      const expectedValue = 'expectedValue'
```

Comment:
Can we change the name of this haha cause this isn't our expected value for this test right? instead its the name of the product

Reply:
e6bc40a8a